src/plugins/dev_tools/public/plugin.ts

```diff
@@ -84,6 +84,20 @@ export class DevToolsPlugin implements Plugin<DevToolsSetup, void> {
   public start() {
     if (this.getSortedDevTools().length === 0) {
       this.appStateUpdater.next(() => ({ navLinkStatus: AppNavLinkStatus.hidden }));
+    } else {
+      this.appStateUpdater.next(() => {
+        const deepLinks: AppSearchDeepLink[] = [...this.devTools.values()]
+          .filter(
+            // Some tools do not use a string title, so we filter those out
+            (tool) => !tool.enableRouting && !tool.isDisabled() && typeof tool.title === 'string'
+          )
+          .map((tool) => ({
+            id: tool.id,
+            title: tool.title as string,
+            path: `#/${tool.id}`,
```

Comment:
Why did you change `DevToolApp.title` from `string` to `ReactNode` to then forcecast it to `string` here?

Reply:
Yeah that wasn't super clear was it. It's because there is a dev tool (Painless Lab) that is passing a component here to be rendered in the tabs UI. So I fixed the type on `DevToolApp` to be accurate and then filtered that dev tool out (for now) and casted this to a string.